charts/jenkins/templates/home-pvc.yaml

```diff
@@ -34,7 +34,7 @@ spec:
 {{- if (eq "-" .Values.persistence.storageClass) }}
   storageClassName: ""
 {{- else }}
-  storageClassName: "{{ .Values.persistence.storageClass }}"
+  storageClassName: "{{ tpl .Values.persistence.storageClass }}"
```

Comment:
Syntax from https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-tpl-function not respected. the `tpl` function take 2 parameters. It should be:
`storageClassName: "{{ tpl .Values.persistence.storageClass . }}"`